test/integration/targets/ec2_vpc_route_table/tasks/main.yml

```diff
@@ -211,24 +236,25 @@
         Public: "true"
         Name: "Public route table"
       <<: *aws_connection_info
+      routes: []
     register: purge_routes
 
   - name: assert purge routes worked
     assert:
       that:
         - purge_routes.changed
-        # FIXME: purge_routes does work but the result is not up to date and returns
-        # the route - a wait period might help
-        # - purge_routes.route_table.routes|length == 1
+        - purge_routes.route_table.routes|length == 1
         - purge_routes.route_table.id == create_public_table.route_table.id
 
+
```

Comment:
Extra whitespace.

Reply:
The extra whitespace was deliberate, using two blank lines to separate each task/assert group. If you don't like that as a style, I can fix it - I don't think I've been hugely consistent at implementing that as a style :)

Reply:
Okay, cool. I don't mind either way, just noticed that was not how the rest of the tasks were spaced. :)

Reply:
Fixed